src/AdoNet/Orleans.Reminders.AdoNet/SiloHostBuilderReminderExtensions.cs

```diff
@@ -59,12 +66,12 @@ public static ISiloHostBuilder UseAdoNetReminderService(this ISiloHostBuilder bu
         /// <returns>
         /// The provided <see cref="ISiloHostBuilder"/>, for chaining.
         /// </returns>
-        public static IServiceCollection UseAdoNetReminderService(this IServiceCollection services, Action<AdoNetReminderTableOptions> configure)
+        private static void UseAdoNetReminderService(this IServiceCollection services, Action<AdoNetReminderTableOptions> configure)
```

Comment:
I believe this is out of the scope of this PR. And we haven't decide to hiding those IServiceCollection extension entirely or not as a team, at least I don't remember we had that discussion. 
Even if we decided to make those extension methods private, then we should do it to all extension methods, so that it is consistent through the team.

Reply:
I'll leave it as public now, but we're going to remove it eventually.